packages/desktop-client/src/components/reports/graphs/CashFlowGraph.tsx

```diff
@@ -41,7 +41,7 @@ function CustomTooltip({ active, payload, isConcise }: CustomTooltipProps) {
     return null;
   }
 
-  const [{ payload: data }] = payload;
+  const [{ payload: data }] = payload ?? [{}];
```

Comment:
Sometimes the cashflow page crashes. I didn't figure out why (it's unrelated to the changes in this PR). But this quick patch fixes the crash.